### DISCUSSION TITLE
Build common news programs in your country ：）

### DISCUSSION THREAD

**Discussion by fwz233** [zepp-health/zeppos-forum#52] [General]
I opened a new project [_Anynews_](https://github.com/fwz233/Anynews)

You can install it here [QRcode](https://fwz233.github.io/fwz233/qr.html?app=2)

I am not familiar with common news websites in English environment, so I chose **the Guardian** New York Times and CNN.

**Please tell me four common news websites in your country, and I can build appropriate news software for your country**

Or you can **fork** my [project](https://github.com/fwz233/Anynews) and try to build your own national news program

**Comment by remco59** [comment]
Here are some news sites in the Netherlands:

NOS: The news show on channel 1, probably the most well known. [nos.nl](https://www.nos.nl) 

RTL Nieuws: The news show on the commercial channel 4. [rtlnieuws.nl](https://www.rtlnieuws.nl) 

NU.nl: mostly an online only news site. [nu.nl](https://www.nu.nl) 


Edit: translation error

**Reply by fwz233** [reply]
I have created a new branch and will update and publish it in the next time
https://github.com/fwz233/Anynews/tree/NetherlandsNews

**Comment by MyoungXUE** [comment]
Great project, can I add to the awesome repository?

https://github.com/zepp-health/awesome-zeppos

**Reply by fwz233** [reply]
可以可以😂

**Comment by wain01** [comment]
Hello, can you please make news for Czech Republic? Below most popular sites:
novinky.cz
idnes.cz
aktualne.cz
ct24.cz

Thank you.

**Reply by fwz233** [reply]
It has been completed. In recent days, I will submit it to the app store, and soon I can install them in the app store

**Comment by Areg472** [comment]
Add Armenian news. Most popular one is azatutyun.am, civilnet.am, news.am and tert.am

**Reply by fwz233** [reply]
Thank you so much

**Comment by Cotrox** [comment]
Italy: RaiNews e TGcom24

**Reply by fwz233** [reply]
Thank you so much
I will use these two news sources to make a new version

**Comment by tumbleweed05** [comment]
I couldn't install on GTR mini (ZeppOS2.0)

**Reply by fwz233** [reply]
Because the online scan code installation version does not add GTR mini to the Appid
I will upload a version later

**Comment by Areg472** [comment]
So. Why can't I uninstall the app from my amazfit band 7?
